app/assets/v2/js/pages/mentors.js

```diff
@@ -0,0 +1,103 @@
+var mentors = (function() {
+
+  var pageIdx = 1;
+  var pageSize = 10;
+
+  var init = function() {
+    $('#save_search, #search_sort').hide();
+    var chekcbox_filter_tmpl = $.templates('#mentor_checkbox_filter');
+
+    var experience_filters = [
+      {
+        id: '1_5',
+        value: '1 - 5'
+      },
+      {
+        id: '5_10',
+        value: '5 - 10'
+      },
+      {
+        id: '10_15',
+        value: '10 - 15'
+      },
+      {
+        id: '15_20',
+        value: '15 - 20'
+      },
+      {
+        id: '20_',
+        value: '20+'
+      }
+    ];
+
+    experience_filters.forEach((filter) => {
+      var html = chekcbox_filter_tmpl.render(filter);
+
+      $('#mentors_experience_filters').append(html);
+    });
+
+    function debounce(func, wait, immediate) {
+      var timeout;
+
+      return function() {
+        var context = this;
+        var args = arguments;
+        var later = function() {
+          timeout = null;
+          if (!immediate)
+            func.apply(context, args);
+        };
+        var callNow = immediate && !timeout;
+
+        clearTimeout(timeout);
+        timeout = setTimeout(later, wait);
+        if (callNow)
+          func.apply(context, args);
+      };
+    }
+
+    $('#keywords').keyup(debounce(function() {
+      $('#mentor_search_results').html('');
+      fetchMentors(1, 10);
+    }, 500));
+
+    $('#load_more').click(function() {
```

Comment:
@kziemianek could we change this to `$( "#load_more" ).on( "click", function() {` as the former has been marked for as to-be retired by jquery